ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -110,6 +110,11 @@ if test -f "${SHARED_DIR}/proxy-conf.sh"; then
     # shellcheck disable=SC1090
     source "${SHARED_DIR}/proxy-conf.sh"
   fi
+  if [[ "${CLUSTER_TYPE}" == "nutanix" ]]; then
```

Comment:
So the nutanix flow still need to run installer destroy behind proxy ? For `SSL_CERT_FILE` block, I think we even should move it out of the line 101 if block.